components/raftstore/src/store/config.rs

```diff
@@ -432,6 +432,14 @@ impl Config {
             ));
         }
 
+        if self.raft_max_size_per_msg.0 == 0
+            || self.raft_max_size_per_msg.0 > ReadableSize::gb(32).0
```

Comment:
Maybe it should be limited to 1GiB.

Reply:
@BusyJay said there are users that set this to `3gb`.

Reply:
What I mean is not larger than 3GiB. The value comes from rocksdb's limitation on value size.

Reply:
done. Sorry, I misunderstood your meaning.